admin/acceptance_tests/conftest.py

```diff
@@ -55,8 +68,8 @@ def app_env():
 @pytest.fixture(scope="session")
 @pytest.mark.usefixtures("app_env", "dbsession")
 def app(app_env, dbsession):
+    del dbsession
     config = testing.setUp(registry=app_env['registry'])
-    config.add_request_method(lambda request: dbsession, 'dbsession', reify=True)
```

Comment:
This was done in purpose, in order to allow tests and test data factory to share same session. Can you please have a look at learn_test.py and check that your approach don't forbid the tests run ? If not, I a am affraid we have to rewrite all the test.

Reply:
I do not know the reason to change this, but I'm pretty sure there is a simple solution to get the expected session in request without changing all the tests code.